old-files/.goreleaser.yml

```diff
@@ -0,0 +1,31 @@
+builds:
```

Comment:
I think you can safely remove those files.
They are part of the history, so we can always retrieve them

Reply:
Done